requirements.txt

```diff
@@ -16,3 +16,4 @@ rich
 seaborn>=0.13.2
 sqlalchemy>=2.0
 typer>=0.12
+networkx
```

Comment:
I would put at least a minimum of the current major.minor version - I know from experience that there have been breaking changes in the networkx API.